_maps/RandomRuins/SpaceRuins/infested_frigate.dmm

```diff
@@ -0,0 +1,5162 @@
+	desc = "A product so good your enemies still own one. The dim bloody light compliments the TV's.";
+	name = "Nanotrasen brand microwave";
+	pixel_y = 7
+	},
+/obj/structure/cable,
+/obj/machinery/chem_dispenser/drinks/fullupgrade{
+	pixel_y = 32
+	},
+/turf/open/floor/mineral/titanium/tiled/white,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"UV" = (
+/obj/effect/decal/cleanable/dirt,
+/obj/structure/alien/egg/burst,
+/obj/item/ammo_casing/spent,
+/obj/item/ammo_casing/spent,
+/turf/open/floor/mineral/plastitanium/red,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"UX" = (
+/obj/machinery/door/airlock/hatch,
+/obj/effect/decal/cleanable/blood{
+	icon_state = "tracks";
+	dir = 4
+	},
+/obj/structure/cable,
+/obj/effect/mapping_helpers/airlock/locked,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/turf/open/floor/mineral/plastitanium,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"Vo" = (
+/obj/structure/cable,
+/obj/effect/turf_decal{
+	icon_state = "warningline_white"
+	},
+/obj/machinery/light/broken/directional/east,
+/obj/effect/turf_decal{
+	icon_state = "warningline_white";
+	dir = 4
+	},
+/obj/machinery/computer/operating,
+/turf/open/floor/pod/dark,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"VQ" = (
+/obj/structure/window/spawner{
+	dir = 8
+	},
+/obj/structure/table/glass,
+/obj/item/surgical_drapes,
+/obj/effect/spawner/random/medical/surgery_tool_advanced,
+/obj/item/reagent_containers/hypospray/medipen{
+	pixel_y = 6
+	},
+/obj/item/reagent_containers/hypospray/medipen,
+/obj/effect/turf_decal{
+	icon_state = "warningline_white"
+	},
+/obj/structure/cable,
+/turf/open/floor/pod/dark,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"VR" = (
+/obj/item/tank/internals/oxygen/empty,
+/turf/open/space/basic,
+/area/template_noop)
+"VU" = (
+/obj/structure/cable,
+/obj/effect/decal/cleanable/vomit/old,
+/obj/item/broken_bottle,
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/turf/open/floor/mineral/plastitanium/red,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"VV" = (
+/obj/structure/cable,
+/obj/structure/barricade/security,
+/obj/structure/door_assembly/door_assembly_hatch{
+	name = "SYN-C Brutus Bridge"
+	},
+/obj/machinery/atmospherics/pipe/smart/manifold4w/supply/hidden,
+/turf/open/floor/mineral/plastitanium/red,
+/area/ruin/space/infested_frigate{
+	has_gravity = 1
+	})
+"Wk" = (
+/obj/structure/alien/weeds,
+/obj/structure/cable,
+/obj/effect/decal/cleanable/glass{
+	icon_state = "plasmatiny"
+	},
+/obj/effect/decal/cleanable/glass{
+	icon_state = "plasmalarge"
+	},
+/obj/item/shard{
+	icon_state = "titaniumtiny"
+	},
+/obj/structure/closet/crate/large{
+	anchored = 1;
+	name = "crew supplies"
+	},
+/obj/effect/spawner/random/aimodule/harmful,
+/obj/effect/spawner/random/contraband,
+/obj/effect/spawner/random/contraband,
+/obj/effect/spawner/random/contraband,
+/obj/effect/spawner/random/contraband,
```

Comment:
I notice you have these spawners inside the map, but /obj/effect/spawner/random/contraband does not spawn anything